src/plugins/texthighlight/_base.scss

```diff
@@ -2,10 +2,8 @@
  * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
  * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
  */
-.wb-txthl {
-	.txthl {
-		background-color: $txthl-bg-color;
-		color: $txthl-color;
-		font-weight: $txthl-font-weight;
-	}
+mark {
```

Comment:
This code would need to be moved into the bootstrap override CSS.

Reply:
@duboisp Resolved!